.github/workflows/build-and-test.yml

```diff
@@ -28,12 +28,15 @@ jobs:
         with:
           sarif_file: service/build/reports/spotbugs/main.sarif
 
-  black:
+  linters:
```

Comment:
Putting this scan in a GitHub action that's not a gradle task makes it harder to run locally. If it were a task, we could add it to `test` so it's run as part of `gradle build`. I don't know how hard this would be to do, so that can wait for a future PR if it's not easy to do now.

Reply:
I checked and there's a shellcheck plugin, and spotless apparently contains black directly. I'm not sure how to configure spotless to use black on python code.

Reply:
@fboulnois I think this is something you added to my PR?

Reply:
I've created a separate PR for setting up gradle to do these scans.